extensions/wikia/Email/SocialLinksGenerator.class.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+namespace Email;
+
+/**
+ * Class SocialLinksGenerator
+ * @package Email
+*/
+class SocialLinksGenerator
+{
+	const SOCIAL_MEDIA = [
+		'en' => [
+			'Facebook' => [
+				'url' => 'https://www.facebook.com/getfandom',
+				'footerIcon' => 'Facebook-Icon-2x.png',
+				'welcomeIcon' => 'Connect-FB.png',
+			],
+			'Twitter' => [
+				'url' => 'https://www.facebook.com/getfandom',
+				'footerIcon' => 'Twitter-Icon-2x.png',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Twitter-Icon-2x.png" 9 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)